solutions/cses-1706.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: cses-1706
+source: CSES
+title: School Excursion
+author: Óscar Garries
```

Comment:
put your name here lol

Reply:
Oh :)) I thought it should be the problem's author name.